templates/api/knowledge-panels/contribution/data_quality_tags.tt.json

```diff
@@ -58,7 +58,7 @@
                 {
                     "element_type": "action",
                     "action_element": {
-                        "html": `[% lang(action_name) %]`,
+                        "html": `[% edq(lang(action_name)) %]`,
```

Comment:
it's not needed here… as we are using back-thick.

I have to check in other places.